CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_twilio_source v0.2.1
```

Comment:
I'd actually suggest we make this a breaking change, as the datatype of a field changing counts as a schema change IMO 

I know that's annoying since we'll have to make a release in the transform package, but perhaps this could potentially break someone's transformation if they're expecting a string... Do you have any thoughts?

Reply:
Thanks Jamie, this is a good call!